apps/web/src/components/healthcare/ClinicalWorkflowPatterns.tsx

```
@@ -0,0 +1,851 @@
+
+                {/* Step Content */}
+                <div className="flex-1 min-w-0">
+                  <div className="flex items-start justify-between">
+                    <div className="flex-1">
+                      <h4 className={cn(
+                        'font-medium mb-1',
+                        isActive ? 'text-[#112031]' : 'text-[#B4AC9C]'
+                      )}>
+                        {step.title}
+                        {step.required && (
+                          <span className="text-red-500 ml-1" aria-label="obrigatório">*</span>
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)